buildifier/def.bzl

```diff
@@ -58,6 +62,10 @@ _buildifier = rule(
             doc = "Linting mode",
             values = ["", "fix", "warn"],
         ),
+        "lint_warnings": attr.string_list(
+            allow_empty = True,
+            doc = "A list of warnings to pass to the linter, prefixed with +/- depending on if you want to include or exclude",
```

Comment:
This is not exactly correct, maybe replace with something like

"all prefixed with +/- if you want to include in or exclude from the default set of warnings, or none prefixed with +/- if you want to override the default set, or "all" for all avaliable warnings"

Reply:
Ah yea, updated with your string